extensions/GitExtension/FileExplorerGitIntegration.UnitTest/WslIntegratorUnitTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using FileExplorerGitIntegration.Models;
+
+namespace FileExplorerGitIntegration.UnitTest;
+
+[TestClass]
+public class WslIntegratorUnitTests
+{
+    [TestMethod]
+    [DataRow("\\wsl$\\Ubuntu-20.04\\home\\user\\repo")]
```

Comment:
All these WSL paths should start with a double `\\`, but this is a single `\`. Seems we should enforce that it begins with a double slash.

Reply:
Resolved in latest. Thanks!